packages/renderer/src/lib/container/ContainerEmptyScreen.svelte

```diff
@@ -24,7 +24,7 @@ onMount(async () => {
       <h1 class="pf-c-title pf-m-lg">No containers</h1>
       <div class="pf-c-empty-state__body">Run a first container using the following command line:</div>
       <div class="flex flex-row bg-gray-800 w-full items-center p-2 mt-2">
-        <div id="noContainerCommandLine">podman run redhat/ubi8-micro echo hello world</div>
+        <div id="noContainerCommandLine">podman run -ti quay.io/podman/hello</div>
```

Comment:
We don't need the -ti flag to echo the output so I would keep it simple

```suggestion
        <div id="noContainerCommandLine">podman run quay.io/podman/hello</div>
```